static/js/gio_calendar_next_cm.js

```diff
@@ -21,7 +21,7 @@ function retrieveNextCommunityMeeting() {
       var shortDescription = e.short_description ? `<p>${e.short_description}</p>` : '';
       var generalLocation = e.location ? `<location><strong>Location:</strong><br>${e.location}</location><hr>` : '';
       var eventLink = e.event_url ? `<p><a href="${e.event_url}"><i class="glyphicon glyphicon-new-window"></i> Event Link</a></p>` : '';
-      var communityMeetingAgenda = e.community_meeting ? ` <a class="btn btn-tnris btn-md" href="https://tnris-org-static.s3.amazonaws.com/documents/GISComm_Agenda_20200114.pdf">Download Agenda</a>` : '';
+      var communityMeetingAgenda = e.community_meeting_agenda_url ? ` <a class="btn btn-tnris btn-md" href=${e.community_meeting_agenda_url}>Download Agenda</a>` : '';
```

Comment:
missing quotes, i think it might break if they aren't there
href="${e.community_meeting_agenda_url}"

Reply:
gotchya i'll fix it